tests/unit/s2n_override_openssl_random_test.c

```diff
@@ -75,4 +115,3 @@ int main(int argc, char **argv)
 }
 
 #endif
```

Comment:
Is there verification that the fixed DRBG cannot be used if you're not in a unit test?

Reply:
Added tests for that and added a check in the generate method to reseed for prediction resistance.